### PR TITLE
cmd/devp2p: less output in node crawler

### PR DESCRIPTION
**Pull request by holiman**
Our discovery crawler spits out a huge amount of logs, most of which is pretty non-interesting. This PR moves the very verbose output to `Debug`, and adds a 8-second status log message giving the general idea about what's going on.

### CODE REVIEW

File: cmd/devp2p/crawl.go [hunk 1]
```diff
@@ -113,12 +140,17 @@ func (c *crawler) runIterator(done chan<- enode.Iterator, it enode.Iterator) {
 	}
 }
 
-func (c *crawler) updateNode(n *enode.Node) {
+// updateNode updates the info about the given node, and returns a status
+// about what changed
+// 0: node skipped
```
Comment (MariusVanDerWijden) [inline cmd/devp2p/crawl.go:145]: `nodeRemoved = iota` comment seems a bit stale

**Review by MariusVanDerWijden — APPROVED** [review]
SGTM

File: cmd/devp2p/crawl.go (hunk 1 above)
Comment (holiman) [reply cmd/devp2p/crawl.go:145]: Yeah I'm just now removing that :)

**Comment by holiman** [comment]
Output example: 

```
INFO [02-13|18:48:01.661] Crawling in progress                     added=0 updated=0 removed=0 ignored(recent)=0 ignored(incompatible)=0
INFO [02-13|18:48:09.661] Crawling in progress                     added=0 updated=0 removed=0 ignored(recent)=0 ignored(incompatible)=0
INFO [02-13|18:48:17.661] Crawling in progress                     added=0 updated=0 removed=0 ignored(recent)=0 ignored(incompatible)=0
INFO [02-13|18:48:25.661] Crawling in progress                     added=0 updated=0 removed=0 ignored(recent)=0 ignored(incompatible)=0
INFO [02-13|18:48:33.661] Crawling in progress                     added=0 updated=0 removed=0 ignored(recent)=0 ignored(incompatible)=0
```

**Pull request merged**